tests/aws/services/events/test_events.py

```diff
@@ -101,7 +101,8 @@ def test_put_events_without_source(self, snapshot, aws_client):
         response = aws_client.events.put_events(Entries=entries)
         snapshot.match("put-events", response)
 
-    @markers.aws.unknown
+    @markers.aws.only_localstack
+    # tests for legacy v1 provider delete once v1 provider is removed, v2 covered in separate tests
     @pytest.mark.skipif(
         is_old_provider(),
```

Comment:
Why do we skip this test for the old provider if it is only intended for it?

Reply:
correct, I validated the test